sklearn/tests/test_docstring_parameters.py

```diff
@@ -277,6 +266,13 @@ def test_fit_docstring_attributes(name, Estimator):
     if Estimator.__name__ == "MDS":
         est.set_params(normalized_stress="auto")
 
+    # Low max iter to speed up tests
+    if "max_iter" in est.get_params():
+        est.set_params(max_iter=2)
```

Comment:
Removing `batch_size=5` for minibatch dict learning makes it slower. So I thought that we should set a low max_iter for all estimators to speed up the test since we're only interested on the existence of fitted attributes